script.js

```diff
@@ -1,12 +1,193 @@
+
+let episodesData = [];
+let showsData = [];
+
+
+ 
+const loadEpisodes = async () => {
+  try {
+    const res = await fetch('https://api.tvmaze.com/shows/82/episodes');
+    
+    episodesData = await res.json();
+    displayEpisodes(episodesData);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+const loadShows = async () => {
+  try {
+    const res = await fetch('  http://api.tvmaze.com/shows/1/episodes');
+    //How do I  get show id here  everytime user select item??
+    //Think !!
+    showsData = await res.json();
+    
+    return showsData;
+
+  } catch(err){
+    console.log(err);
+  }
+}
+
+
+
+//const allEpisodesDetails = setup();
+searchBar.addEventListener('keyup', (e) => {
+
+  const searchString = e.target.value.toLowerCase();
+  const filteredMovies = episodesData.filter((item) => {
+    return (
+      item.name.toLowerCase().includes(searchString) ||
+      item.summary.toLowerCase().includes(searchString)
+    );
+  });
+  displayEpisodes(filteredMovies);
+});
+
+function displayShows(Shows) {
+  let displayShow = Shows.map((item) => {
+    return `<article class='movie-item'>
+            <img src=${item.image.medium} alt=${item.name} class='photo'/>
+            <div class='item-info'>
+            <header>
+            <h4>${item.name}</h4>
+            <h4 class ='episode-code'> ${'S0' + item.id} </h4>
+            </header>
+            <p class='item-text'> ${item.summary}'
+            </p>
+            <div>
+            </article>`;
+   
+  });
+  displayShow = displayShow.join('');
+  sectionCenter.innerHTML = displayShow;
 }
 
-function makePageForEpisodes(episodeList) {
+
+
+function displayEpisodes(Movies) {
+  let totalMovies = Movies.length;
+  let total = document.querySelector(".total")
+  total.textContent = totalMovies;
+ 
+   let displayEpisode = Movies.map((item)=> {
+
+
+      return `<article class="movie-item">
+              <img src=${item.image.medium} alt=${item.name} class="photo" />
+            <div class="item-info">
+                  <header>
+                    <h4>${item.name}</h4>
+
+                  <h4 class="episode-code">${'S0'+item.season.toString()+'E0'+item.number.toString()}</h4>
+                  </header>
+                <p class="item-text">
+                   ${item.summary}
+                </p>
+               </div>
+
+             </article>`;
+     });
+
+     displayEpisode = displayEpisode.join(" ");
+  
+     sectionCenter.innerHTML = displayEpisode;
+}
+//create select box for episodes
+getAllEpisodes().forEach((item) => {
+
+  let opt = 'S0' + item.season.toString() + 'E0' + item.number.toString() + '-' + item.name;
+
+  let el = document.createElement("option");
+  el.textContent = opt;
+  el.value = opt;
+  selectEpisodes.appendChild(el);
+
+});
+   
+//select box logic 
+  
+
+  document.querySelector("#select-episode").addEventListener('change', (e) => {
+    const selectString = e.target.value;
+    console.log(selectString);
```

Comment:
Be sure to remove `console.logs` when from your code when you've finished with them